tests/strategies/relative_orders_test.py

```diff
@@ -0,0 +1,215 @@
+from dexbot.strategies.relative_orders import Strategy
+from tests.fixtures import fixture_data_RO
+import logging
+import math
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s %(funcName)s %(lineno)d  : %(message)s'
+)
+
+
+class Test_Strategy:
```

Comment:
Why you aren't using pytest native fixture-based approach? We most-likely will have problems later with it. Take a look at my PR https://github.com/Codaone/DEXBot/pull/531/files